jenkinsfile-examples/android-build-flavor-from-branch/JenkinsFile

```diff
@@ -15,14 +15,14 @@ node {
   echo "Building flavor ${flavor}"
 
   //build your gradle flavor, passes the current build number as a parameter to gradle
-  sh "./gradlew clean assemble${flavor}Debug -PBUILD_NUMBER=${env.BUILD_NUMBER}"
+  sh './gradlew clean assemble${flavor}Debug -PBUILD_NUMBER=${BUILD_NUMBER}'
```

Comment:
`${flavor}` isn't an environment variable (`BRANCH_NAME` is) so this won't work. Use `withEnv`.